src/utils/getSortedPeople.ts

```diff
@@ -0,0 +1,34 @@
+import { Person } from '../types';
+
+export const getSortedPeople = (
+  people: Person[],
+  sort: string,
+  order: string,
+) => {
+  const sortedPeoople = [...people];
+
+  switch (sort) {
+    case 'name':
+      sortedPeoople.sort((a, b) => a.name.localeCompare(b.name));
+      break;
+
+    case 'sex':
+      sortedPeoople.sort((a, b) => a.sex.localeCompare(b.sex));
+      break;
+
+    case 'born':
+      sortedPeoople.sort((a, b) => a.born - b.born);
+      break;
+
+    case 'died':
+      sortedPeoople.sort((a, b) => a.died - b.died);
+      break;
```

Comment:
Maybe we can combine this four cases in two cases, >> sortedPeople.sort(a,b => a[sort].localeCompare(b[sort]), but it's just comment.

Reply:
agree.
good point. tanks